go/oprf/oprf.go

```diff
@@ -239,36 +240,43 @@ func (c Client) Setup(ciphersuite string, pogInit gg.PrimeOrderGroup) (Participa
 // Blind samples a new random blind value from ZZp and returns P=r*T where T is
 // the representation of the input bytes x in the group pog.
 func (c Client) Blind(x []byte) (gg.GroupElement, *big.Int, error) {
+	P, _, r, err := c.BlindInternal(x)
+	return P, r, err
+}
+
+// Blind samples a new random blind value from ZZp and returns P=r*T where T is
```

Comment:
Rewrite comment to reference `BlindInternal`

Reply:
Fixed!